catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/resource/download/ReliableResourceDownloadManager.java

```diff
@@ -67,24 +74,12 @@ public void init() {
 
     }
 
-    public void cleanUp() {
```

Comment:
This should probably be replaced since Blueprint expects it. Although if you've already accounted for that, my apologies.